reproducibility_project/lrc_shift_subproject/src/engines/gromacs/project.py

```diff
@@ -329,7 +326,7 @@ def _setup_mdp(fname, template, data, overwrite=False):
             )
 
     rendered = template.render(data)
-    with open(fname, "w") as f:
+    with open(fname, "w"):
         f.write(rendered)
 
```

Comment:
Maybe im missing something, but dont you need to have the `as f` section to use the `f.write` command on line 330?